### PR TITLE
Add missing comma, fix formatting and unmute diagnostics test

### PR DESCRIPTION
**Pull request by mrnugget**
## High-level description

This PR unmutes and fixes a once flaky test which has been muted because of its flakyness. But since it's been muted other bugs have been introduced that did not lead to a failed test. 

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4531](https://jira.mesosphere.com/browse/DCOS_OSS-4531) test_dcos_diagnostics.test_dcos_diagnostics_bundle_download_and_extract assertionerror test bundle download and validate zip file

## Related tickets (optional)

Other tickets related to this change:

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by mrnugget** [comment]
@mesosphere-mergebot label Ready for Review

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -330,11 +330,6 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
 
 
-@pytest.mark.xfailflake(
-    jira='DCOS-44935',
-    reason='test_dcos_diagnostics.test_dcos_diagnostics_bundle_create_download_delete is Flaky',
-    since='2018-11-20',
-)
```
[the remark this answers is not in the document]
Comment (mrnugget) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Do you mean the enterprise tests? Or just all the tests here in this PR?

[the remark this answers is not in the document]
Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Yep, Entreprise or maybe there is a way to mute test only for EE? @adamtheturtle

[the remark this answers is not in the document]
Comment (adamtheturtle) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: There is a way to skip tests in Enterprise.

See https://github.com/dcos/dcos/blob/597a349b2d1f71062030f819cacdf4c4cec3d177/packages/dcos-integration-test/extra/test_legacy_user_management.py#L37 as an example of a test file doing this.

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Pull request merged**